package.json

```diff
@@ -18,7 +18,9 @@
   },
   "homepage": "https://github.com/cherryontech/squad-debugging-darlings#readme",
   "dependencies": {
+    "axios": "^1.3.4",
     "bcryptjs": "^2.4.3",
+    "cors": "^2.8.5",
     "dotenv": "^16.0.3",
```

Comment:
We shouldn't be having any package.json files outside of the backend/frontend folders. Delete this and the lock file and have your depencies installed within back-end itself